assets/js/post_office_search.tsx

```diff
@@ -13,25 +13,28 @@ const root = createRoot(elem);
 const { locationsSearchUrl } = elem.dataset;
 
 root.render(
-  <FullAddressSearch
-    disabled={false}
-    handleLocationSelect={null}
-    locationsURL={locationsSearchUrl}
-    noInPersonLocationsDisplay={NoInPersonLocationsDisplay}
-    onFoundLocations={() => {}}
-    registerField={() => {}}
-    resultsHeaderComponent={() => (
-      <Alert type="info" className="margin-bottom-4">
-        <strong>
-          {t('in_person_proofing.body.location.po_search.you_must_start.message', {
-            app_name: 'Login.gov',
-          })}
-        </strong>{' '}
-        <a href={t('in_person_proofing.body.location.po_search.you_must_start.link')}>
-          {t('in_person_proofing.body.location.po_search.you_must_start.link_text')}
-        </a>
-      </Alert>
-    )}
-    usStatesTerritories={UsStatesTerritories}
-  />,
+  <form>
```

Comment:
I think this looks good overall! Simple fix. Maybe add a `novalidate` attribute to the form element to bring it into alignment with what is happening on the IDP side and since we're doing JS validation and error handling?

Reply:
Good recommendation.  Resolved with commit 441e258436239cd12af26bacb788d59516fe1fbc